templates/repo/diff/box.tmpl

```diff
@@ -130,9 +130,9 @@
 									<span class="ui label">{{$.locale.Tr "repo.diff.vendored"}}</span>
 								{{end}}
 								{{if and $file.Mode $file.OldMode}}
-									<span class="gt-ml-4 gt-mono">{{$file.OldMode}} → {{$file.Mode}}</span>
+									<span class="gt-ml-4 gt-mono">{{$.locale.Tr ($file.ModeTranslationKey $file.OldMode)}} → {{$.locale.Tr ($file.ModeTranslationKey $file.Mode)}}</span>
```

Comment:
In RTL languages, would the arrow flip automatically?

Reply:
I have no idea.
You tell me.
That functionality exists already in main.
It depends on how Unicode works in that regard.

Reply:
I think hebrew/arabic is RTL?
If that's true, I just tested how it will look like in hebrew/arabic for the screenshot above: (I don't speak those languages, so I simply used a translation above, no idea what it currently says there)
![grafik](https://github.com/go-gitea/gitea/assets/51889757/15c08095-ca56-4dfc-a9e3-a24ad06ffe7e)
Is that understandable for RTL speakers?

Reply:
Ah wait, even I can see that the arrow points in the wrong direction.

Reply:
That's more how it is supposed to look like, correct?
![grafik](https://github.com/go-gitea/gitea/assets/51889757/eac6b447-c683-41b8-b74d-14a4c6bc7178)

Reply:
Better, I can't confirm because I don't know Arabic (I do know Hebrew) but you understood what I was going for, thank you.

Reply:
So, isn't this enhancement then technically even a bugfix? :thinking:
That problem already exists in main…

Reply:
In main it's not internationalized so it's ok, no?

Reply:
Ah, LTR content is automatically displayed as LTR?
That's not a pagewide setting?
That sounds really confusing to read.

Reply:
I just meant that there's an assumption anything that's not using `Tr` is only readable in English